org.civicrm.reqangular/src/common/services/api.js

```diff
@@ -84,6 +85,9 @@ define([
           }.bind(this))(),
           (function () {
             var params = _.assign({}, filters, { 'return': 'id' });
```

Comment:
Use the chaining feature of lodash (also add an inline comment to explain what's going on)
```js
(function () {
  // Removing chained calls, they are not necessary for getting the count
  var params = _({})
    .assign(filters, { 'return': 'id' })
    .omit(function (__, key) { return _.startsWith(key, 'api.'); })
    .value();

  return this.sendGET(entity, action, params, cache);
}.bind(this))()
```